configure.ac

```diff
@@ -6,6 +6,8 @@ dnl libtool-2.4.2
 
 AC_INIT([log4cplus],[1.2.0])
 AC_CANONICAL_TARGET
+m4_include([m4/ax_check_host_and_build_os.m4])
```

Comment:
The usual way of writing `configure.ac` at least in log4cplus is to create a stand alone macro and put it into a `.m4` file in `m4` directory, then to include that file in `acinclude.m4`, then to use that macro inside `configure.ac`.